src/content/release-notes.mdx

```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.2
+
+16 August 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.30
+
+### Bug Fixes
+
+- Fix deploys for repositories with a trailing slash
```

Comment:
Should we clarify that this is for the catalog?

Reply:
It would happen also if they deploy it from the CLI for example. It was detected with the catalog, but it might happen from other places